tests/chrono_binding.cpp

```diff
@@ -0,0 +1,41 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <catch2/catch.hpp>
+
+using namespace sqlite_orm;
+
+#if defined(SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED)
```

Comment:
Need to test for `#if __cpp_lib_chrono >= 201907L && __cpp_lib_format >= 202110L` here, too.

Reply:
but  I did already!

Reply:
not in tests/chrono_binding.cpp. I think the diff is clearer than our conversation ;)

Reply:
where are you seeing this?

Reply:
ok found'

Reply:
this file should be deleted!